test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -419,7 +421,12 @@ public void testAclSqlResultSetHasNonOverlappingTwoRecord()
         + GsaSpecialColumns.GSA_DENY_GROUPS + ","
         + GsaSpecialColumns.GSA_DENY_USERS + ") values "
         + "('pgroup1, pgroup2', 'puser1, puser2', "
-        + "'dgroup1, dgroup2', 'duser1, duser2'), "
+        + "'dgroup1, dgroup2', 'duser1, duser2')");
+    executeUpdate("insert into acl("
```

Comment:
For posterity, these would have made a good separate CL. It's hard to scan for other changes here.